### ISSUE TITLE
href path issue for files under views folder 

### ISSUE THREAD

**Issue by lambone** [ashwin01/ReactJS-AdminLTE#3]
I find out that the path in the html files under views folder are broken, for example, ReactJS-AdminLTE/reactjs-adminlte/views/index2.html , 

<link rel="stylesheet" href="dist/css/AdminLTE.min.css">

but dist dir is under public folder. Either the path is not correct or these files under views folder need to be move to public.

**Comment by ashwin01** [comment]
Did you open views/index2.html directly in chrome, or did you open it by running the web server?

The files in views folder are meant for getting served from a web server. So if you run -
    
    node server.js

and go to [http://localhost:8080/index2.html](http://localhost:8080/index2.html), you will see the page getting served with all paths pointing to the correct resources. The idea of putting them in views folder is to show that this is server-side rendering (node server is configured to look in the views folder for serving pages), which could be done on the client side in reactjs, like views/widgets.html. You can see that there is almost no markup in widgets.html.

Does this clarify your issue?

**Comment by lambone** [comment]
yes, I exactly follow the instruction. go to http://localhost:8080/index2.html
You can reproduce by click the "documentation", the link broken.
![image](https://cloud.githubusercontent.com/assets/4221783/11774793/f8c1098a-a273-11e5-9b44-044a67f9a12a.png)

**Comment by ashwin01** [comment]
Ah yes, I see what you mean. The links don't work because the file paths differ while being served from the web-server. 

Will fix it soon, thanks for pointing it out!
